### PR TITLE
BXMSPROD-1025 build chain centralize build configuration

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1025

* https://github.com/kiegroup/lienzo-core/pull/135
* https://github.com/kiegroup/lienzo-tests/pull/110
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1489
* https://github.com/kiegroup/kie-soup/pull/160
* https://github.com/kiegroup/appformer/pull/1063
* https://github.com/kiegroup/droolsjbpm-knowledge/pull/478
* https://github.com/kiegroup/drools/pull/3177
* https://github.com/kiegroup/optaplanner/pull/978
* https://github.com/kiegroup/jbpm/pull/1783
* https://github.com/kiegroup/kie-jpmml-integration/pull/45
* https://github.com/kiegroup/droolsjbpm-integration/pull/2290
* https://github.com/kiegroup/openshift-drools-hacep/pull/101
* https://github.com/kiegroup/droolsjbpm-tools/pull/121
* https://github.com/kiegroup/kie-uberfire-extensions/pull/107
* https://github.com/kiegroup/kie-wb-playground/pull/99
* https://github.com/kiegroup/kie-wb-common/pull/3457
* https://github.com/kiegroup/drools-wb/pull/1433
* https://github.com/kiegroup/optaplanner-wb/pull/382
* https://github.com/kiegroup/jbpm-designer/pull/892
* https://github.com/kiegroup/jbpm-work-items/pull/163
* https://github.com/kiegroup/jbpm-wb/pull/1466
* https://github.com/kiegroup/kie-docs/pull/2947
* https://github.com/kiegroup/optaweb-employee-rostering/pull/513
* https://github.com/kiegroup/optaweb-vehicle-routing/pull/381
* https://github.com/kiegroup/kie-wb-distributions/pull/1066

* https://github.com/kiegroup/github-action-build-chain/pull/82
* https://github.com/kiegroup/build-chain-configuration-reader/pull/1

### CODE REVIEW

File: .ci/project-dependencies.yaml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    dependencies:
+      - project: kiegroup/drools
+      - project: kiegroup/kie-soup
+
+  - project: kiegroup/optaplanner
+    dependencies:
+      - project: kiegroup/drools
+        mapping:
+          source: 7.x
+          target: master
```
Comment (mbiarnes) [inline .ci/project-dependencies.yaml:42]: @Ginxo what for is the target here? Only a question for comprehension.

**Review by almope — APPROVED** [review]

File: .ci/project-dependencies.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      - project: kiegroup/jbpm-designer
+      - project: kiegroup/jbpm-work-items
+
+  - project: kiegroup/kie-docs
+
+  - project: kiegroup/optaweb-employee-rostering
+    dependencies:
+      - project: kiegroup/optaplanner
+        mapping:
+          source: master
```
Comment (mareknovotny) [inline .ci/project-dependencies.yaml:126]: I believe optaweb's repos are now on the same mapping as Optaplanner. So here should be 7.x

File: .ci/project-dependencies.yaml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+          target: 7.x
+
+  - project: kiegroup/optaweb-vehicle-routing
+    dependencies:
+      - project: kiegroup/optaplanner
+        mapping:
+          source: master
```
Comment (mareknovotny) [inline .ci/project-dependencies.yaml:133]: I believe optaweb's repos are now on the same mapping as Optaplanner. So here should be 7.x

File: .ci/project-dependencies.yaml (hunk 2 above)
Comment (mareknovotny) [reply .ci/project-dependencies.yaml:126]: look here https://github.com/kiegroup/optaweb-employee-rostering/tree/7.x

File: .ci/project-dependencies.yaml (hunk 3 above)
Comment (mareknovotny) [reply .ci/project-dependencies.yaml:133]: look here https://github.com/kiegroup/optaweb-vehicle-routing/tree/7.x

**Review by mbiarnes — APPROVED** [review]
+1

**Pull request merged**